WMFModel/WMFArticle+CoreDataProperties.h

```diff
@@ -0,0 +1,29 @@
+#import "WMFArticle+CoreDataClass.h"
+
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface WMFArticle (CoreDataProperties)
+
++ (NSFetchRequest<WMFArticle *> *)fetchRequest;
+
+@property (nonatomic) BOOL isBlocked;
+@property (nullable, nonatomic, copy) NSString *key;
```

Comment:
Is this the url? If so, should we just call it URL? 

If not, can we add the URL?